### PR TITLE
ENT-4295: Fixed an issue while parsing ps output on AIX

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

**Comment by olehermanse** [comment]
@cf-bottom jenkins with exotics.

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/1715/)

(with exotics)

https://ci.cfengine.com/job/pr-pipeline/1715/

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/1716/)

(with exotics)

https://ci.cfengine.com/job/pr-pipeline/1716/

**Comment by olehermanse** [comment]
AIX only:

[Build Status](https://ci.cfengine.com/job/pr-pipeline/1717/)

**Comment by olehermanse** [comment]
@vpodzime please review

File: libpromises/processes_select.c [hunk 1]
```diff
@@ -795,8 +814,8 @@ Solaris 9:
             return false;
         }
 
-        // If zombie, check if field is empty.
-        if (pca == PCA_ZombieSkipEmptyColumns && zombie)
+        // If zombie/exiting, check if field is empty.
```
Comment (vpodzime) [inline libpromises/processes_select.c]: Would be nice to update the huge comment at the beginning of the function.

**Comment by Lex-2008** [comment]
approved on be½ of @vpodzime:
> Looks good to me

File: libpromises/processes_select.c (hunk 1 above)
Comment (vpodzime) [reply libpromises/processes_select.c]: Thanks!

**Comment by vpodzime** [comment]
> @vpodzime I tried updating the comment. Please re-review.

Looks good to me. :+1:

**Pull request merged**